### PR TITLE
ensureutf8 on all strings that are passed to tracer

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: proxy/proxy.go [hunk 1]
```diff
@@ -1489,7 +1489,7 @@ func (p *Proxy) setCommonSpanInfo(u *url.URL, r *http.Request, s ot.Span) {
 		setTag(s, HTTPPathTag, u.Path).
```
Comment (AlexanderYastrebov) [inline proxy/proxy.go:1489]: I guess this was missing

**Comment by AlexanderYastrebov** [comment]
:+1:

File: proxy/proxy.go (hunk 1 above)
Comment (szuecs) [reply proxy/proxy.go:1489]: I tested that if you parse as we do with url.Parse all attributes are encoded correctly (path,query,hostname), the same for NewRequest(), so I don't know. I will try to remove the setTag encoder to test another assumption

**Comment by szuecs** [comment]
:+1:

**Pull request merged**